![](https://github.com/codebiogenomics/codebiogenomics/blob/main/codebiogenomics.jpg)

I work at Instytut Chemii Bioorganicznej Polskiej Akademii Nauk, Poznań Poland as a part of a very good team. I work as area expert and responsible for genomic analysis and software development from illumina and long read sequencing. I work using Python, R, Julia, RUST, Typescript and I provide  bio-software development, algorithms, bioinformatics, machine learning  as an Area Expert area for Illumina and LongRead sequencing technologies  across all domains of life covering plant, bacterial, fungal and humans.

Scientific and Bio-Computing Areas of Interest: 
🧬 I am specific to the genomics area such as RNA-seq, single cell genomics, and any genomics or protein language models and biological deep bio ML 🧬 and bio-software. 
❌ I don't do protein structural modelling, and ML on the same.
❌ What i don't code anymore: C++ since 2021 (2010-2021), dropped DevOPS (2024). 

🧬 Sequencing specifics: \
2010-2021: Plant, Bacterial, Fungi 🧬 RNASeq, GenomSeq, Phylogenomics, PacBio Sequencing, Single Cell Analysis, Bio-software. 👨‍💻 \
2021:2023: Machine Learning, Bio-software. 🧬 \
2024: PanGenome, Bio-software. \
2025: Human Genomics 🧬and Bio-software, Mobile Bio-software 🧬 

🧬 Language stack: I am very specific and oritented toward taking typed languages over the others. My language stack has been dominated by the typed languages and you can find below:
Software Development: C++(Cropped 2021) and started RUST. 
Web Development: Only Typescript and React. 
Bioinformatics Data Analysis: Python, R and Julia. 
Mobile Development: ReactNative and Expo.

[![My Skills](https://skillicons.dev/icons?i=js,html,css,bash,rust,materialui,obsidian,r,py,pytorch,tensorflow,tauri,react,ts,vscode,remix)](https://skillicons.dev)
